cws-installer/src/main/java/jpl/cws/task/CwsInstaller.java

```diff
@@ -1064,33 +1066,90 @@ private static void setupLogstash() {
 
 	private static void setupElasticsearch() {
 
```

Comment:
Make sure curl is done after the URL is checked

Reply:
https://github.com/NASA-AMMOS/common-workflow-service/blob/83f251dcb3c79ed16351d77d1739928a314b9cac/cws-installer/src/main/java/jpl/cws/task/CwsInstaller.java#L1934